docs/content/gameobject/api-workerconnector.md

```diff
@@ -16,14 +16,14 @@ During the creation of your workers, they attempt to connect to the SpatialOS Ru
 | OnWorkerCreationFinished | Action<Worker> | A callback that is invoked when the worker was created successfully. |
 
 
-** Methods **
+**Methods**
 
 ```csharp
 public async Task Connect(string workerType, ILogDispatcher logger)
 ```
 Parameters:
   * `string workerType`: The type of the worker
```

Comment:
Needs a newline at the start of the list

Reply:
Otherwise it won't render properly

Reply:
ah missed that one